test-app/src/main/java/org/readium/r2/testapp/reader/preferences/UserPreferences.kt

```diff
@@ -63,7 +63,8 @@ private fun <P : Configurable.Preferences<P>, E : PreferencesEditor<P>> UserPref
         Text(
             text = title,
             textAlign = TextAlign.Center,
-            style = MaterialTheme.typography.h6,
+            style = MaterialTheme.typography.headlineSmall,
+            color = MaterialTheme.colorScheme.onSurface,
```

Comment:
Isn't this already the default color for a `Text`? Usually you wouldn't need to customize the default `color`.

Reply:
I'll get a screenshot, but it was using black text on a dark brown background for this specifically.

Reply:
You would think.
![Screenshot_20240205_210821](https://github.com/readium/kotlin-toolkit/assets/8315038/e18dd6af-f68b-43d1-9185-ed01cea6c848)

Reply:
Ha yes, I encountered the same issue in Aldiko. I found out that you need to embed everything in a `Surface` composable using Material 3 in order for dark mode to work correctly.

I think you could do this around the `Content()` here:  https://github.com/readium/kotlin-toolkit/blob/1570e80ab0a8f28255e53cabb291b1152f565a5f/test-app/src/main/java/org/readium/r2/testapp/utils/compose/ComposeBottomSheetDialogFragment.kt#L35

Or maybe in `AppTheme` directly, but there are some places where we might not want to use a `Surface`.

Reply:
Wrapped this around a `Surface`.